coderd/userauth.go

```diff
@@ -342,3 +469,66 @@ func (api *API) userOIDC(rw http.ResponseWriter, r *http.Request) {
 	}
 	http.Redirect(rw, r, redirect, http.StatusTemporaryRedirect)
 }
+
+// githubLinkedID returns the unique ID for a GitHub user.
+func githubLinkedID(u *github.User) string {
+	return strconv.FormatInt(u.GetID(), 10)
+}
+
+// oidcLinkedID returns the uniqued ID for an OIDC user.
+// See https://openid.net/specs/openid-connect-core-1_0.html#ClaimStability.
```

Comment:
This doesn't link correctly because of the period at the end.

Reply:
colin it's a sentence, I'm a man of punctuation

Reply:
what's the point of the link if it doesn't work!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

Reply:
damn straight j. learn up coolin